bigbluebutton-html5/imports/ui/components/nav-bar/settings-dropdown/component.jsx

```diff
@@ -79,16 +80,52 @@ class SettingsDropdown extends Component {
   }
 
   componentWillMount() {
-    const { intl, isFullScreen, mountModal } = this.props;
+    this.createMenu();
+  }
+
+  componentWillReceiveProps(nextProps) {
+    this.setState(
+      { isFullScreen: nextProps.isFullScreen },
+      () => this.createMenu(),
```

Comment:
Instead of creating the entire menu every time `componentWillReceiveProps`  runs, it would be better to only update the menu item that needs updating.